index.md

```diff
@@ -152,6 +152,7 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
 | 2 | ref/hooks | [Hooks](https://discourse.ubuntu.com/t/hooks/28555)|
 | 2 | ref/perf-benchmarks | [Performance benchmarks](https://discourse.ubuntu.com/t/performance-benchmarks/24709)|
 | 2 | ref/webrtc-streamer | [WebRTC streamer](https://discourse.ubuntu.com/t/webrtc-streamer/30195)|
+| 2 | ref/license-information | [License information](tbd)
```

Comment:
I was trying to see what other projects at Canonical (or at large) were calling these pages, but it seems like none of the other Canonical projects have similar pages in their docs.

Some projects call it "Third-party licenses", but they are usually available verbatim in that case, e.g.:
https://developer.spotify.com/third-party-licenses

Reply:
Since the same company is much involved in the components (Snaps, Charms, LXD), I am a bit sceptical to call it third-party. Ideally this is "License information for software used by Anbox Cloud" but we are constrained by the TOC as we cannot have long titles in the TOC. I can use it for the page title though.

So my suggestion is to keep it as "License information" for the TOC and when creating the discourse post, we can use "License information for related software" or "License information for software components used". What do you think?

Reply:
I think "License information for software components used" is good, but it would be nice to know what the others think about this.

Reply:
I would keep it simple as `License information`